server/src/main/java/io/druid/server/coordination/ZkCoordinator.java

```diff
@@ -299,13 +308,12 @@ public void loadLocalCache()
       try {
         final DataSegment segment = jsonMapper.readValue(file, DataSegment.class);
           if (segmentManager.isSegmentCached(segment)) {
-          cachedSegments.add(segment);
+            cachedSegments.add(segment);
```

Comment:
I think this is still weird with spacing.  IT could be github, but it appears as if the `if` is 2 spaces ahead of it's paired `else` a line below.